include/wlr/types/wlr_screencopy_v1.h

```diff
@@ -27,9 +27,14 @@ struct wlr_screencopy_manager_v1 {
 	void *data;
 };
 
+struct wlr_screencopy_connection_v1 {
+	struct wlr_screencopy_manager_v1* manager;
+	struct wl_list damages;
+};
+
 struct wlr_screencopy_frame_v1 {
 	struct wl_resource *resource;
-	struct wlr_screencopy_manager_v1 *manager;
+	struct wlr_screencopy_connection_v1 *connection;
```

Comment:
The frame can outlive the client/connection object. We need to handle this somehow.